modules/asio/1.21.0/presubmit.yml

```diff
@@ -0,0 +1,12 @@
+matrix:
+  platform:
+  - centos7
+  - debian10
+  - ubuntu2004
+  - macos
+  - windows
+tasks:
+  verify_targets:
+    name: Verify build targets
+    platform: ${{ platform }}
+    build_targets: []
```

Comment:
Can you specify a build target here to make sure this module actually works?
probably:
```
build_targets:
  -- @asio//:asio
```